bicep/files-to-load/util/cluster_inits_slurm_ccw.txt

```diff
@@ -0,0 +1,6 @@
+
+        [[[cluster-init cyclecloud/slurm:default]]]
+        Optional = true
+
+        [[[cluster-init cyclecloud/ccw:default]]]
```

Comment:
Extra cluster-inits should be added using Ryan's new additional Projects feature (even if they are added automatically and not visible in the Marketplace UI)